adapter/slack.py

```diff
@@ -112,3 +113,22 @@ async def list_bots(self, request: Request):
             return Response(status_code=200)
         except SlackApiError as e:
             raise HTTPException(status_code=400, detail=f"Slack API Error : {e}")
+    
+    def event_message(self, event, say):
+        if 'thread_ts' in event:
+            self.event_message_replied(event, say)
+    
+    def event_message_replied(self, event, say):
+        parent_message = self.app.client.conversations_history(
+            channel=event['channel'],
+            inclusive=True,
+            oldest=event['thread_ts'],
+            limit=1
+        )
+        parent_user, parent_text = parent_message['messages'][0]['user'], parent_message['messages'][0]['text']
+        
+        if parent_user == self.app_user_id and 'asked' in parent_text:
+            self.bot_replied(event, say)

```

Comment:
Does this mean that the bot can only  reply the OP of the thread? What happens if another user asks in the same thread?

Reply:
Okay, that's good